vms/rpcchainvm/vm.go

```diff
@@ -43,20 +43,11 @@ func Serve(ctx context.Context, vm block.ChainVM, opts ...grpcutils.ServerOption
 			fmt.Println("vm server: graceful termination success")
 		}()
 
-		for {
-			select {
-			case s := <-signals:
-				switch s {
```

Comment:
We only handle `SIGINT` and `SIGTERM` and both are handled the same way so no need for switch